ui/views/controls/native/native_view_host.cc

```diff
@@ -127,6 +127,23 @@ void NativeViewHost::VisibilityChanged(View* starting_from, bool is_visible) {
   Layout();
 }
 
+void NativeViewHost::OnSoftVisibilityChanged(bool visible) {
+  if (!native_view_ || !native_wrapper_.get())
```

Comment:
I would keep it - as it is on 3 other places like that if ... native_wrapper_.get()